oaknational/Oak-Web-Application
# Feat/host mathjax package json LESQ-386

## Description

- Adds MathJax node package
- Adds copy-webpack-plugin node package
- Update next.config (Copies mathjax from node to public)
- Updates provider to point to public folder rather than cdn (default)

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2007--oak-web-application.netlify.thenational.academy/teachers/programmes/maths-secondary-ks4-foundation/units/surds/lessons/accuracy-of-final-answers#exit-quiz
You should see mathjax in the quizzes


## Checklist
